charts/rancher-vsphere-csi/values.yaml

```diff
@@ -145,37 +145,37 @@ global:
 # https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/3.0/vmware-vsphere-csp-getting-started/GUID-D4AAD99E-9128-40CE-B89C-AD451DA8379D.html#kubernetes-versions-compatible-with-vsphere-container-storage-plugin-1
 versionOverrides:
   # Versions from https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/release-3.0/manifests/vanilla/vsphere-csi-driver.yaml
-  - constraint: ">= 1.24 < 1.28"
+  - constraint: ">= 1.24 < 1.29"
     values:
       csiController:
         image:
           repository: rancher/mirrored-cloud-provider-vsphere-csi-release-driver
-          tag: v3.0.2
+          tag: v3.1.2
           csiAttacher:
             repository: rancher/mirrored-sig-storage-csi-attacher
-            tag: v4.2.0
+            tag: v4.4.2
```

Comment:
It's v4.3.0 upsteam - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/v3.1.2/manifests/vanilla/vsphere-csi-driver.yaml#L246

Any specific reason we're going with higher version or am I looking in the wrong place?